ContainsDuplicate/Program.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ContainsDuplicate
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Duplicates duplicates = new Duplicates();
+
+            // Case 1
+            int[] numbers1 = { 1, 2, 3, 1 };
+            Console.WriteLine(duplicates.CheckForDuplicates(numbers1));
```

Comment:
always try to add (assign) the return data into the variable and then pass the variable to another method, why? because it's better to debugging and clean to read the code